pkg/kv/kvserver/rditer/testdata/TestSelect/r1

```diff
@@ -1,23 +1,23 @@
 echo
 ----
-Select({ReplicatedBySpan:{/Min-c} ReplicatedByRangeID:false UnreplicatedByRangeID:false}):
+Select({ReplicatedBySpan:{/Min-c} ReplicatedSpansFilter:0 ReplicatedByRangeID:false UnreplicatedByRangeID:false}):
```

Comment:
It would be good to extend this test to have `ReplicatedSpansFilter` other than "include all".

Reply:
Done.